File: app/src/main/java/com/example/pondibrian/ukulimaiq/PriceActivity.java
package com.example.pondibrian.ukulimaiq;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.ViewGroup;
import android.widget.ListView;

public class PriceActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_price);
        ListView listView = findViewById(R.id.listv);
        ViewGroup headerView = (ViewGroup)getLayoutInflater().inflate(R.layout.header,listView,false);
        listView.addHeaderView(headerView);
        String[] items =getResources().getStringArray(R.array.list_items);
        ListAdapter adapter = new ListAdapter(this,R.layout.row_layout,R.id.txtmodel,items);
        listView.setAdapter(adapter);
    }
}

File: app/src/main/kotlin/com/pondipb/pondibrian/ukulimaiq/PriceActivity.kt
package com.pondipb.pondibrian.ukulimaiq

import android.os.Bundle
import android.view.ViewGroup
import android.widget.ListView
import androidx.appcompat.app.AppCompatActivity

class PriceActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_price)
        
        val listView: ListView = findViewById(R.id.listv)
        val headerView = layoutInflater.inflate(R.layout.header, listView, false) as ViewGroup
        listView.addHeaderView(headerView)
        
        val items = resources.getStringArray(R.array.list_items)
        val adapter = ListAdapter(this, R.layout.row_layout, R.id.txtmodel, items)
        listView.adapter = adapter
    }
}
